x11/windowmaker/Portfile

```diff
@@ -1,8 +1,10 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
```

Comment:
This is our standard modeline which states that indentations occur at multiples of 4 spaces, but the portfile's existing whitespace does not conform to that. Either don't add the modeline, or make your changes in two separate commits: one that adds the modeline and adjusts the whitespace to conform, and a second that makes functional changes.